core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.component.ts

```diff
@@ -77,6 +77,7 @@ export class StateInteractionEditorComponent
   directiveSubscriptions = new Subscription();
   hasLoaded: boolean;
   interactionEditorIsShown: boolean;
+  showToggleBtn: boolean = true;
```

Comment:
I noticed that we assigned a `true` value here and upon initialization, we assign it a `false` value. Is this intentional?

Reply:
my mistake it is false everywhere.